.github/workflows/on-pull-request.yml

```diff
@@ -29,7 +29,7 @@ jobs:
           ${{ runner.OS }}-
 
     - name: Install dependencies ⏬
-      run: npm ci
+      run: npm ci --force
```

Comment:
Why is this needed?

Reply:
`ol-mapbox-style` allows only for `ol` `<= 7.5.1`, but we have `7.5.2`...

Reply:
Alright, could you please add a comment to this line?